com.unity.render-pipelines.universal/Shaders/2D/Light2D-Point-Volumetric.shader

```diff
@@ -49,18 +49,18 @@ Shader "Hidden/Light2d-Point-Volumetric"
 
             TEXTURE2D(_FalloffLookup);
             SAMPLER(sampler_FalloffLookup);
-            float _FalloffIntensity;
+            half _FalloffIntensity;
 
             TEXTURE2D(_LightLookup);
             SAMPLER(sampler_LightLookup);
-            float4 _LightLookup_TexelSize;
+            half4 _LightLookup_TexelSize;
```

Comment:
If this is power of two texture size if should be fine, otherwise you should double check you have enough precision to store in half.